contracts/token/ERC721/ERC721Pausable.sol

```diff
@@ -4,10 +4,20 @@ import "./ERC721.sol";
 import "../../utils/Pausable.sol";
 
 /**
- * @title ERC721 Non-Fungible Pausable token
- * @dev ERC721 modified with pausable transfers.
+ * @dev ERC721 token with pausable token transfers, minting and burning.
+ *
+ * Useful for scenarios such as preventing trades until the end of an evaluation
+ * period, or having an emergency switch for freezing all token transfers in the
+ * event of a large bug.
```

Comment:
I feel like we should have a "See {Pausable}" link, and in that page have this kind of explanation.

Reply:
It'd be good to have ideas in Pausable, but I'd still add specific rationale whenever we add Pausable to some other contract.